### ISSUE TITLE
[BUG] Timer sound

### ISSUE THREAD

**Issue by fabian-bickel** [0blu/Timer#5]
I fixed teh timer-sound.sp with:

replace this:
if (FileExists(fileSound) && g_bTimerWorldRecord)

with:
if (FileExists(fileSound))

**Comment by 0blu** [comment]
Ok, but it would be better if you make a merge request.

**Comment by fabian-bickel** [comment]
yeah but we have so much edit in there wih some fixes and not all is fixed is will do it later and make a big merge request
